contracts/metatx/MinimalForwarder.sol

```diff
@@ -31,6 +31,11 @@ contract MinimalForwarder is EIP712 {
 
     mapping(address => uint256) private _nonces;
 
+    /**
+     * @dev Signature does not match `ForwardRequest` provided.
+     */
+    error MinimalForwarderInvalidSignature(address from, uint256 nonce);
```

Comment:
This was my intention initially as mentioned in [this convo](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4261#discussion_r1204582686), however, I didn't end up liking the idea because we can potentially end up with N different `Invalid*` errors for each thing that may be invalid from the request.

I'm still open to keeping it but I'd rather define a way of making a generic invalid error here if possible

Reply:
Ok, I went ahead with the suggestion but with a few considerations. Here's the [commit](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4261/commits/c796d15d5fa7924ef1dbabbd8e772f612d9847c0).

The errors defined where:

```solidity
error MinimalForwarderInvalidSigner(address signer, address from);
error MinimalForwarderInvalidNonce(address signer, uint256 current);
```

The rationale of the arguments is according to EIP-6093, first _who_ and then _what_. I think this is even more helpful because the `requestNonce` and `requestFrom` are in the function parameters (same for `from` in `InvalidSigner` but is just inverting the order)